### PR TITLE
Flush v0.13 deprecation backlog

### PR DESCRIPTION
**Pull request by balopat**
Flush v0.13 deprecation backlog - so we don't forget to do it before v0.13 release.

### CODE REVIEW

File: cirq-core/cirq/sim/act_on_args.py [hunk 1]
```diff
@@ -213,22 +212,6 @@ def __iter__(self) -> Iterator[Optional['cirq.Qid']]:
     def _act_on_fallback_(self, action: Any, qubits: Sequence['cirq.Qid'], allow_decompose: bool):
         """Handles the act_on protocol fallback implementation."""
 
-    @property  # type: ignore
```
Comment (daxfohl) [inline cirq-core/cirq/sim/act_on_args.py:216]: I think we should remove it from the constructor here too since it's no longer used anywhere.

**Review by MichaelBroughton — APPROVED** [review]
LGTM.

**Comment by MichaelBroughton** [comment]
Gentle reminder @balopat

**Comment by MichaelBroughton** [comment]
This seems to be underwater with merge conflicts. Going to close for and re-assign to myself.

**Pull request closed without merging**